app/src/main/java/framgia/com/ichat/data/model/PrivateRoom.java

```diff
@@ -0,0 +1,54 @@
+package framgia.com.ichat.data.model;
+
+import java.util.HashMap;
+
+public class PrivateRoom {
+    private String mId;
+    private String mName;
+    private HashMap<String,Message> mMessages;
+    private String image;
```

Comment:
Please check convention 2.1 Java Style Rule
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md#21-java-style-rules
Non-public, non-static field names start with m.
```suggestion
    private String mImage;
```